backend/app/rest/api/rest.go

```diff
@@ -354,13 +358,17 @@ func (s *Rest) controllerGroups() (public, private, admin, rss) {
 
 	pubGrp := public{
 		dataService:      s.DataService,
+		searchService:    nil,
 		cache:            s.Cache,
 		imageService:     s.ImageService,
 		commentFormatter: s.CommentFormatter,
 		readOnlyAge:      s.ReadOnlyAge,
 		webRoot:          s.WebRoot,
 	}
-
+	if s.SearchService != nil {
+		// we can't assign without checking for nil because we can get not nil interface containing nil underline
```

Comment:
Sounds strange, would be nice to avoid this check if possible.

Reply:
It's about this feature, when we assign null pointer to struct to interface

https://play.golang.org/p/OS8a9wbG0dJ